api/server/api.go

```diff
@@ -145,14 +145,14 @@ func (s *rulesetAPI) eval(w http.ResponseWriter, r *http.Request, path string) {
 	}
 
 	if err != nil {
-		if err == regula.ErrRulesetNotFound {
+		if err == errortype.ErrRulesetNotFound {
```

Comment:
What about `errors` instead of `errortype`?

Reply:
Done